images/custom-error-pages/rootfs/main.go

```diff
@@ -97,6 +97,19 @@ func main() {
 	http.ListenAndServe(fmt.Sprintf(":8080"), nil)
 }
 
+func findAcceptedFormat(requestedMimes string) (string, error) {
+	// get request format from Accept header
+	for _, requestedMime := range strings.Split(requestedMimes, ",") {
+		requestedMime = strings.TrimSpace(requestedMime)
+		exts, err := mime.ExtensionsByType(requestedMime)
+		if err != nil || len(exts) == 0 {
+			continue
+		}
+		return exts[0], nil
```

Comment:
Can we assume the first value is always the right value?

Reply:
I implemented a function that retrieves available extensions and updated this one to pick into this generated list instead of the first one.